src/metabase/sync/sync_metadata/fields.clj

```diff
@@ -264,14 +337,23 @@
   ([database :- i/DatabaseInstance, table :- i/TableInstance]
    (sync-util/with-error-handling (format "Error syncing fields for %s" (sync-util/name-for-logging table))
      (let [db-metadata (db-metadata database table)]
+
+       (update! fields-per-table-hist (count db-metadata))
```

Comment:
What are the average number of fields per table

![fields-per-table](https://user-images.githubusercontent.com/215910/34177219-771a774c-e4c8-11e7-97a1-da6cd61e4523.png)